roles/deploy_flask_app/files/vars/main.yaml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
I'm a little confused about the structure of this role. Is there a reason the `vars` folder is under the `files` folder? Is the idea that these are example vars one would provide when running the role? If so, that should be explained in the README and each var in here should be documented.

Reply:
In the README, I have referenced the files under `roles/deploy_flask_app/files` as sample files. However, I can provide more details if needed. Additionally, since the user may have a different setup, do you think it's necessary to document the variables used in a setup sample task? As for the files themselves, they were included to provide extra assistance, but they are not actually required.